### PR TITLE
update(docs): more detail about container runtime populated k8s fields

### PR DESCRIPTION
**Pull request by vjjmiras**
Signed-off-by: Vicente J. Jiménez Miras <vjjmiras@gmail.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

> /kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

> /area documentation

> /area videos

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by incertum** [comment]
Thanks a bunch 🙏 ! Great concise changes!

File: content/en/docs/reference/rules/supported-fields/index.md [hunk 1]
```diff
@@ -206,6 +206,14 @@ Name | Type | Description
 `container.liveness_probe` | CHARBUF | The container's liveness probe. Will be the null value ("N/A") if no liveness probe configured, the liveness probe command line otherwise
 `container.readiness_probe` | CHARBUF | The container's readiness probe. Will be the null value ("N/A") if no readiness probe configured, the readiness probe command line otherwise
 
+{{% alert color="primary" %}}
+
+The [k8s set of fields](#field-class-k8s) `k8s.ns.name` and `k8s.pod.*` (i.e., `k8s.pod.name`, `k8s.pod.id`, `k8s.pod.labels`, and `k8s.pod.label.*`) are populated with data fetched from the **container runtime**.
+
+Therefore, they can also be accessed without having the **Kubernetes Audit Events** gathering functionality enabled.
```
Comment (jasondellaluce) [inline content/en/docs/reference/rules/supported-fields/index.md:213]:
```suggestion
Therefore, they can also be accessed without having the **Kubernetes Metadata Enrichment** functionality enabled (`-k` Falco option).
```

File: content/en/docs/reference/rules/supported-fields/index.md [hunk 2]
```diff
@@ -286,8 +294,15 @@ Name | Type | Description
...
 ### Field Class: k8s
 
-Kubernetes related context. When configured to fetch from the API server, all fields are available. Otherwise, only the `k8s.pod.*` and `k8s.ns.name` fields are populated with data gathered from the container runtime.
+Kubernetes related context.
+
+{{% alert color="warning" %}}
+
+The set of fields `k8s.ns.name` and `k8s.pod.*` (e.g., `k8s.pod.name`) are populated with data fetched from the **container runtime**, so they don't require the **Kubernetes Audit Events** gathering functionality to be enabled.
+
+All the other fields will only be available when Falco also fetches the [Kubernetes Audit Events](/docs/event-sources/kubernetes-audit/) from the Kubernetes API server.
```
Comment (jasondellaluce) [inline content/en/docs/reference/rules/supported-fields/index.md:303]:
```suggestion
The set of fields `k8s.ns.name` and `k8s.pod.*` (e.g., `k8s.pod.name`) are populated with data fetched from the **container runtime**, so they don't require the **Kubernetes Metadata Enrichment** functionality to be enabled.

All the other fields will only be available when Falco also fetches the **Kubernetes Metadata** from the Kubernetes API server  (`-k` Falco option) .
```

**Review by jasondellaluce — COMMENTED** [review]
Just left some suggestions. There here we're mixing the two distinct concepts of K8S metadata enrichment and k8s audit logs:
- K8S metadata is collected by the Falco bundled k8s client by connecting to the API server, and allows extracting the `k8s.*` fields for the `syscall` event source
- K8S Audit Logs are available in Falco as a separate event source `k8s_audit` implemented by the `k8saudit` plugin (and its EKS variant, as of today), and have nothing to do with the `k8s.*` syscall-compatible fields

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**